packages/webpack/lib/plugins/log.js

```diff
@@ -37,8 +46,10 @@ const formatSuccess = (name, duration, assets, isRebuild) => {
 };
 
 exports.LoggerPlugin = class LoggerPlugin {
-  constructor(logger) {
-    this.logger = logger;
```

Comment:
I would personally prefer separate args instead of an options hash to keep this consistent with other APIs, but I am fine with this as it is.